modules/@apostrophecms/storybook/template/.storybook/webpack.js

```diff
@@ -28,15 +28,22 @@ module.exports = async (config) => {
           loader: 'sass-loader',
           options: {
             // sass-loader 9 changed this option name yet again
-            additionalData: `
-              @import 'Modules/@apostrophecms/storybook/scss/storybook';
-            `
+            additionalData:
+`@import "Modules/@apostrophecms/ui/scss/mixins/import-all.scss";
+`
           }
         }
       ]
     }
   );
 
+  // Allow for an npm linked apostrophe module to work
+  config.resolve.modules = [
```

Comment:
isn't this in another branch and PR?

Reply:
It is, but I mentioned that; it's necessary here too to be able to verify the correctness of this new PR for storybook. I am not sure why it shows up as a change still since this is merged in 3.0.

Reply:
OK I merged the 3.0 branch into this branch and as you can see this is no longer an issue.